bower.json

```diff
@@ -44,7 +44,9 @@
     "angular-local-storage": "~0.5.2",
     "angular-promise-extras": "~0.1.8",
     "ng-kaltura-player": "~1.2.1",
-    "pdf.js-viewer": "~1.6.211"
+    "pdf.js-viewer": "~1.6.211",
+    "angular-material": "^1.1.6",
+    "angular-aria": "1.5.11"
```

Comment:
please also add `"angular-messages": "1.5.11"` (seems to be another new requirement)